worldguard-bukkit/src/main/java/com/sk89q/worldguard/bukkit/listener/WorldBlockListener.java

```diff
@@ -66,44 +66,15 @@
  *
  * @author sk89q
  */
-public class WorldGuardBlockListener implements Listener {
-
-    private WorldGuardPlugin plugin;
+public class WorldBlockListener extends AbstractListener {
```

Comment:
what? why rename?

Reply:
The reason is to match the rest of the classes which extend AbstractListener. Only the old listeners which implement all the listener stuff themselves use the `WorldGuard....Listener` format, so I updated it to match the rest of the listeners.